p2p/peerserver/banpeerhandler.go

```diff
@@ -56,9 +61,15 @@ func (sp *serverPeer) addBanScore(persistent, transient uint32, reason string) {
 	if score > warnThreshold {
```

Comment:
I think ban threshold definitions should be refactored and stay together with ban score definitions. So that code reader can be clear to know the `defaultBanThreshold` is 100, and `warnThreshold` is 50, and the `SeriousScore` is 100. and more maintainable for future changes